grails-app/services/org/pih/warehouse/data/DataService.groovy

```diff
@@ -175,7 +175,7 @@ class DataService {
                     }
                 }
 
-                addInventoryLevelToProduct(product, location.inventory, preferredBinLocation, row.minQuantity, row.reorderQuantity, row.maxQuantity, row.preferredForReorder)
+                addInventoryLevelToProduct(product, location.inventory, preferredBinLocation, row.minQuantity, row.reorderQuantity, row.maxQuantity, row.expectedLeadTimeDays, row.replenishmentPeriodDays, row.preferredForReorder)
```

Comment:
The amount of parameters is getting out of hand. Can you pass the entire row here instead of destructured parameters?